packages/features/bookings/lib/handleCancelBooking.ts

```diff
@@ -510,14 +515,6 @@ async function handler(req: CustomRequest) {
         apiDeletes.push(deleteMeeting(videoCredential, uid));
       }
     }
-    // For bookings made before this refactor we go through the old behaviour of running through each video credential
```

Comment:
This change has over 8 months so I say it's good to say removing it will be the right thing to do!